{"pagelen": 100, "values": [{"changes": {"state": {"new": "open", "old": "new"}}, "links": {"self": {"href": "data/repositories/levitsky/pyteomics/issues/47/changes/54482696.json"}, "html": {"href": "#!/levitsky/pyteomics/issues/47#comment-54482696"}}, "issue": {"links": {"self": {"href": "data/repositories/levitsky/pyteomics/issues/47.json"}}, "type": "issue", "id": 47, "repository": {"links": {"self": {"href": "data/repositories/levitsky/pyteomics.json"}, "html": {"href": "#!/levitsky/pyteomics"}, "avatar": {"href": "data/bytebucket.org/ravatar/{3f6302de-10da-4d3c-b258-0bb6a2368972}ts=74456"}}, "type": "repository", "name": "pyteomics", "full_name": "levitsky/pyteomics", "uuid": "{3f6302de-10da-4d3c-b258-0bb6a2368972}"}, "title": "non-ascii characters in AuditCollection lead to UnicodeEncodeError"}, "created_on": "2019-10-22T14:13:11.101823+00:00", "user": {"display_name": "Lev Levitsky", "uuid": "{eb44325f-4ee0-4e0b-a27c-f2ea23122a56}", "links": {"self": {"href": "https://api.bitbucket.org/2.0/users/%7Beb44325f-4ee0-4e0b-a27c-f2ea23122a56%7D"}, "html": {"href": "https://bitbucket.org/%7Beb44325f-4ee0-4e0b-a27c-f2ea23122a56%7D/"}, "avatar": {"href": "data/secure.gravatar.com/avatar/a2593c44c42429c503d2e5e9e307e241d=httpsavatar-management--avatars.us-west-2.prod.public.atl-paas.netinitialsLL-6.png"}}, "nickname": "levitsky", "type": "user", "account_id": "557058:986c547b-c50a-40b3-948a-29b4a93b7b30"}, "message": {"raw": "Can confirm with last release and development versions on Python 2.\n\nThe suggested fix doesn't seem right for Python 3, though. We need to apply the \"ensure string\" logic somewhere, probably in or around `_handle_param()`.", "markup": "markdown", "html": "<p>Can confirm with last release and development versions on Python 2.</p>\n<p>The suggested fix doesn't seem right for Python 3, though. We need to apply the \"ensure string\" logic somewhere, probably in or around <code>_handle_param()</code>.</p>", "type": "rendered"}, "type": "issue_change", "id": 54482696}, {"changes": {"state": {"new": "resolved", "old": "open"}}, "links": {"self": {"href": "data/repositories/levitsky/pyteomics/issues/47/changes/55038535.json"}, "html": {"href": "#!/levitsky/pyteomics/issues/47#comment-55038535"}}, "issue": {"links": {"self": {"href": "data/repositories/levitsky/pyteomics/issues/47.json"}}, "type": "issue", "id": 47, "repository": {"links": {"self": {"href": "data/repositories/levitsky/pyteomics.json"}, "html": {"href": "#!/levitsky/pyteomics"}, "avatar": {"href": "data/bytebucket.org/ravatar/{3f6302de-10da-4d3c-b258-0bb6a2368972}ts=74456"}}, "type": "repository", "name": "pyteomics", "full_name": "levitsky/pyteomics", "uuid": "{3f6302de-10da-4d3c-b258-0bb6a2368972}"}, "title": "non-ascii characters in AuditCollection lead to UnicodeEncodeError"}, "created_on": "2019-11-30T19:49:21.044794+00:00", "user": {"display_name": "Lev Levitsky", "uuid": "{eb44325f-4ee0-4e0b-a27c-f2ea23122a56}", "links": {"self": {"href": "https://api.bitbucket.org/2.0/users/%7Beb44325f-4ee0-4e0b-a27c-f2ea23122a56%7D"}, "html": {"href": "https://bitbucket.org/%7Beb44325f-4ee0-4e0b-a27c-f2ea23122a56%7D/"}, "avatar": {"href": "data/secure.gravatar.com/avatar/a2593c44c42429c503d2e5e9e307e241d=httpsavatar-management--avatars.us-west-2.prod.public.atl-paas.netinitialsLL-6.png"}}, "nickname": "levitsky", "type": "user", "account_id": "557058:986c547b-c50a-40b3-948a-29b4a93b7b30"}, "message": {"raw": null, "markup": "markdown", "html": "", "type": "rendered"}, "type": "issue_change", "id": 55038535}], "page": 1, "size": 2}